app/index.js

```diff
@@ -1,26 +1,19 @@
+import { hot, setConfig } from 'react-hot-loader';
 import React from 'react';
 import { render } from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
 import App from './App';
 import { configureStore } from './redux/configureStore';
 
+const configOptions: $Shape<Object> = {
+  showReactDomPatchNotification: false
+};
+
+setConfig(configOptions);
+
 const store = configureStore();
 
 const renderWrapper = (Component) => {
-  render(
-    <AppContainer>
-      <Component store={store} />
-    </AppContainer>,
-    document.getElementById('root')
-  );
+  render(<Component store={store} />, document.getElementById('root'));
 };
 
-renderWrapper(App);
-
-if (module.hot) {
-  module.hot.accept('./App', () => {
-    // eslint-disable-next-line global-require
-    const NextApp = require('./App').default;
-    renderWrapper(NextApp);
-  });
-}
+renderWrapper(process.env.NODE_ENV === 'development' ? hot(module)(App) : App);
```

Comment:
simplify this, there is no need in wrapping function

Reply:
ok